src/webview/tests.py

```diff
@@ -252,3 +262,16 @@ def test_add_maintainer_invalid_github_handle(self) -> None:
         self.assertEqual(
             response.context["error_msg"], "Could not fetch maintainer from GitHub"
         )
+
+    def test_add_maintainer_widget_visible_when_admin(self) -> None:
+        url = reverse("webview:drafts_view")
+        response = self.client.get(url)
+        response_content = response.content.decode("utf-8")
+        self.assertIn("maintainer-add-container", response_content)
```

Comment:
I'm slightly uneasy about checking rendered response content against a string that we define in the template, but everything else I can come up with is more complicated. So it's probably fine.